### ISSUE TITLE
Unexpected behavior with the country code rule in 2.3

### ISSUE THREAD

**Issue by oschwald** [Respect/Validation#1446]
As of e2b6138b, the following code no longer throws an exception on a lower-cased country code:

```php
<?php

require 'vendor/autoload.php';

use Respect\Validation\Validator as v;

$a = v::keySet(v::key('country', v::countryCode(), false));

$a->check(['country' => 'ca']);
```

In 2.2.4, a lower-cased country code would throw an exception with the error `Uncaught country must be a valid country`.

**Comment by henriquemoody** [comment]
Wouldn't accepting `ca` as a country code be considered a bug? I haven't read any document about it, but I've always seen those codes in uppercase.

**Comment by oschwald** [comment]
Yes, correct. It now accepts lowercase. Before 2.3, it did not accept it.

**Comment by henriquemoody** [comment]
Thank you for reporting 🐼 

I will fix this one as soon as possible!

**Comment by henriquemoody** [comment]
The fix is available on version 2.3.2

Thank you for reporting 🐼
